content/spin/v2/http-outbound.md

```diff
@@ -99,25 +99,23 @@ You can find a complete example of using outbound HTTP in the JavaScript SDK rep
 
 {{ startTab "Python"}}
 
-> [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-python-sdk/spin_http.html)
+> [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-python-sdk/http.html)
```

Comment:
I am 99% sure that the URLs are not used in blog posts because, we just recently even published a documentation site for it. 

It has just been moved behind a `v1` subdirectory. 
https://fermyon.github.io/spin-python-sdk/v1/spin_http.html